feature-contributors/src/main/java/io/github/droidkaigi/confsched2022/feature/contributors/Contributors.kt

```diff
@@ -59,7 +58,7 @@ fun Contributors(
             )
         }
     ) { innerPadding ->
-        Box(modifier = Modifier.padding(innerPadding)) {
+        Box(modifier = modifier) {
             when (uiModel.state) {
                 is Error -> TODO()
                 Loading -> Box(
```

Comment:
I think the loading mark should be centered along with the other screens, so I prefer to add window insets padding here as well!

Reply:
@Mori-Atsushi @takahirom change: [912214e](https://github.com/DroidKaigi/conference-app-2022/pull/461/commits/912214efa44a1e6e78facb8d2ce93e431b9ba8b1)  I have no idea except passing innerPadding to the loading indicator. What do you think? If you have any further nice solutions please tell me.

Reply:
That looks good to me!

Reply:
I think we should always pass modifiers.
We are having the lint for it. So, for now, it is ok. If you want to fix this. You can make a PR!
https://chris.banes.dev/always-provide-a-modifier/